### PR TITLE
feat: Upgrade stripe checkout to latest version

### PR DESCRIPTION
**Pull request by pc-beast**
Fixes https://github.com/fossasia/open-event-server/issues/8146

#### Checklist

- [ ] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [ ] My branch is up-to-date with the Upstream `development` branch.
- [ ] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by mariobehling** [comment]
Could you solve the lint issue, please?

**Comment by codedsun** [comment]
@mariobehling this is currently work in progress.

**Review by codedsun — APPROVED** [review]

**Comment by pc-beast** [comment]
@mariobehling please approve

File: package.json [hunk 1]
```diff
@@ -172,7 +172,8 @@
   },
   "private": true,
   "dependencies": {
-    "ua-parser-js": "^1.0.2"
+    "@stripe/stripe-js": "^1.20.3",
+    "ua-parser-js": "^1.0.1"
```
Comment (maze-runnar) [inline package.json:176]: Why this downgrade was needed?

Comment (pc-beast) [reply package.json:176]: This got lost in some conflict resollution. reverting it.

**Review by maze-runnar — APPROVED** [review]

**Comment by maze-runnar** [comment]
Merging to be tested on test eventyay

**Pull request merged**